config/webpack/webpack.config.renderer.dev.dll.js

```diff
@@ -6,8 +8,9 @@ import webpack from 'webpack';
 import path from 'path';
 import merge from 'webpack-merge';
 import baseConfig from './webpack.config.base';
-import { dependencies } from './package.json';
 
+const pkg = require(path.resolve(process.cwd(), 'package.json'));
```

Comment:
I have not removed the dependencies import its just instead of using `../../package.json`, I preferred using the path from the current working directory and relating `package.json` to it, so that wherever this file maybe we will not have to change the import statement again and again.